app/src/main/java/org/simple/clinic/patient/sync/PatientPayload.kt

```diff
@@ -62,35 +59,24 @@ data class PatientPayload(
     val businessIds: List<BusinessIdPayload>,
 
     @Json(name = "recorded_at")
-    val recordedAt: Instant?
+    val recordedAt: Instant
 ) {
 
   fun toDatabaseModel(newStatus: SyncStatus): Patient {
     return Patient(
         uuid = uuid,
         addressUuid = address.uuid,
         fullName = fullName,
-        searchableName = nameToSearchableForm(fullName),
         gender = gender,
         dateOfBirth = dateOfBirth,
-        age = age?.let {
-          Age(age, ageUpdatedAt!!, computedDateOfBirth(age, ageUpdatedAt))
-        },
+        age = age?.let { Age(age, ageUpdatedAt!!) },
         status = status,
         createdAt = createdAt,
         updatedAt = updatedAt,
         deletedAt = deletedAt,
-        // recordedAt should be changed here when PatientPayload starts receiving this field from server
```

Comment:
The server now sends the `recordedAt` property in all patient payloads, so we no longer have to retain it as a nullable property and add a fallback.